client/src/components/Projects/ColumnHeaderPopups/TextPopup.js

```diff
@@ -4,8 +4,11 @@ import Button from "../../Button/Button";
 import RadioButton from "../../UI/RadioButton";
 import "react-datepicker/dist/react-datepicker.css";
 import { MdClose } from "react-icons/md";
+import ReactSelect from "react-select";
```

Comment:
Long story: The styling of drop-downs was inconsistent across browsers and operating systems (Issue #1785). @bilalbg was not able to fix it with just CSS and resorted to introducing the react-select library to replace the standard react <select> component and creating a custom react component UniversalSelect which wrapped react-select and applied styles to get the appearance the designers wanted. Unfortunately, we dropped the ball on testing his PR, and though the list populated correctly, the selected value would not display the selected value, and this broke every page in the app with drop-downs.  I temporarily switched everything back to <select> to get through Tuesday's stakeholder meeting, but then looked into why the UniversalSelect wasn't working and created Issue #1817 and PR #1818 to fix it, and switched the existing usages back to <UniversalSelect>.  I revised the value and onChange props in UniversalSelect to work like they do in the <select> component. This may or may not be  the best option, but I was just trying to get it working for the usages that were in the app. At this time, we want all the drop-downs to use the UniversalSelect component, so we get the consistent styling. We can discuss if you want.

Reply:
Thank you @entrotech for fixing UniversalSelect.  The value and onChange props work fine, but since UniversalSelect does not accept a `placeholder` prop, I cannot place the search Icon next to the placeholder text.  Please advise.